cfme/tests/services/test_rest_services.py

```diff
@@ -5,12 +5,17 @@
 import fauxfactory
 import pytest
 
+from manageiq_client.api import ManageIQClient as MiqApi
+
+from cfme.rest.gen_data import _creating_skeleton
+from cfme.rest.gen_data import blueprints as _blueprints
+from cfme.rest.gen_data import copy_role
 from cfme.rest.gen_data import dialog as _dialog
-from cfme.rest.gen_data import service_data as _service_data
+from cfme.rest.gen_data import groups
+from cfme.rest.gen_data import orchestration_templates as _orchestration_templates
 from cfme.rest.gen_data import service_catalogs as _service_catalogs
+from cfme.rest.gen_data import service_data as _service_data
 from cfme.rest.gen_data import service_templates as _service_templates
```

Comment:
Thanks, I'll do it in some other PR. I'm happy this one passed PRT and don't want to re-rest :) I'm touching this file in other PRs anyway.